src/index.html

```diff
@@ -1,13 +1,601 @@
+                    alt="Bose portable Smart speaker"
+                  >
+                  </div>
+                </a>
+
+                <h3 class="recommend__name">
+                  <a href="#" class="link">
+                    Bose portable Smart speaker
+                  </a>
+                </h3>
+
+                <p class="recommend__type">
+                  Smart home
+                </p>
+
+                <p class="recommend__price">
+                  $ 399.00
+                </p>
+              </div>
+            </div>
+
+            <div class="
+              recommended__item
+              grid__item
+              grid__item--tab-2-5
+              grid__item--desk-5-8"
+            >
+              <div class="recommend">
+                <a href="#">
+                  <div
+                    data-aos="zoom-out-down"
+                    data-aos-duration="1200"
+                  >
+                    <img
+                    class="recommend__photo"
+                    src="images/photos/3.png"
+                    alt="Bose portable Smart speaker"
+                  >
+                  </div>
+                </a>
+
+                <h3 class="recommend__name">
+                  <a href="#" class="link">
+                    SoundLink Flex Bluetooth speaker
+                  </a>
+                </h3>
+
+                <p class="recommend__type">
+                  Portable bluetooth
+                </p>
+
+                <p class="recommend__price">
+                  $ 149.00
+                </p>
+              </div>
+            </div>
+
+            <div class="
+              recommended__item
+              grid__item
+              grid__item--tab-2-5
+              grid__item--desk-9-12"
+            >
+              <div class="recommend">
+                <a href="#">
+                  <div
+                    data-aos="zoom-out-left"
+                    data-aos-duration="1200"
+                  >
+                    <img
+                    class="recommend__photo"
+                    src="images/photos/4.png"
+                    alt="Bose portable Smart speaker"
+                  >
+                  </div>
+                </a>
+
+                <h3 class="recommend__name">
+                  <a href="#" class="link">
+                    SoundLink Color Bluetooth
+                    speaker II
+                  </a>
+                </h3>
+
+                <p class="recommend__type">
+                  Portable bluetooth
+                </p>
+
+                <p class="recommend__price">
+                  $ 129.00
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="categories">
+        <div class="container">
+          <h2 class="page__section-title">
+            Browse Bose
+            <br>
+            products by category
+          </h2>
```

Comment:
Don't mix different blocks. It's ok with `container` and `grid`, but `page__section-title` needs to be a part of the gallery. To reuse style create a placeholder for such header and extend f.e. the `gallery__title` element. Or create an independent block for the title.